test/ConfigFactory/getConfig.test.ts

```diff
@@ -40,6 +56,16 @@ describe('FacilitatorOptionParser.getConfig()', () => {
     return spy;
   }
 
+  function spyGatewayConfigFromFile(gatewayConfig: any): any {
```

Comment:
Please remove any type

Reply:
There are still many `any` types 🤔

Reply:
for method creating stub should return `SinonStubStatic`.

Reply:
Done